apps/src/templates/sectionProgress/detail/VirtualizedDetailView.jsx

```diff
@@ -220,10 +236,50 @@ class VirtualizedDetailView extends Component {
         )}
       </div>
     );
-  };
+  }
 
-  studentCellRenderer = (studentStartIndex, stageIdIndex, key, style) => {
-    const {section, levelsByLesson, stageExtrasEnabled} = this.props;
+  studentCellRenderer(studentStartIndex, stageIdIndex, key, style) {
+    const {section, levelProgressByStudent, scriptData} = this.props;
+    const student = section.students[studentStartIndex];
+    let child;
+    if (stageIdIndex < 0) {
```

Comment:
What does stageIdIndex < 0 represent?

Reply:
column -1 is the student list (in this legacy component it wasn't a separate component like it is in ProgressTableStudentList in the refactored table). fwiw, this component will be removed once we make the switchover